content/en/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Check whether Dockershim deprecation affects you
```

Comment:
Is the focus of this page to:
- help cluster administrators determine if their cluster is using Docker and `dockershim`?
- help cluster administrators add/configure another container runtime?
- provide steps to update running applications that may be tightly integrated with Docker?

Reply:
Intention was this:

> - provide steps to update running applications that may be tightly integrated with Docker?

Good question, perhaps this would mean the better place for this doc would be one level up (not under cluster administration)